runner/test/cases/server/config.test.js

```diff
@@ -83,6 +83,7 @@ suite(`Server Config`, () => {
       isSandbox: false,
     };
 
+    console.log(buildConfig);
```

Comment:
Not sure if leaving this in was intentional?

Reply:
oops, didn't catch that on another PR... do you mind if I open a separate PR so I'm not waiting for the smoke tests..?

Reply:
never mind, this is out of date with master, I'm going to have to run them anyway